core/src/main/scala/io/finch/package.scala

```diff
@@ -15,6 +15,7 @@ package object finch extends Endpoints
     sealed abstract class RequestItem(val kind: String, val nameOption:Option[String] = None) {
       val description = kind + nameOption.fold("")(" '" + _ + "'")
     }
+    final case class AttributeItem(name: String) extends RequestItem("attribute", Some(name))
```

Comment:
Why do you think we have to introduce a new request item for attributes? We've been using `ParamItem` until now and I actually think it's perfectly reasonable.

Reply:
Well, my idea behind it is that they are different entities, so different items. But if you have a strong opinion on that, I could remove it

Reply:
My personal preference is to keep using `ParamItem` just b/c

- We've been using it before and
- I really want to see us removing `RequestItem` thing all-together at some point (and possibly replacing with something else)